[2024-02-11T00:47:47.040+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: censoEnade.download_conceito_2021 manual__2024-02-11T00:47:41.823216+00:00 [queued]>
[2024-02-11T00:47:47.056+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: censoEnade.download_conceito_2021 manual__2024-02-11T00:47:41.823216+00:00 [queued]>
[2024-02-11T00:47:47.057+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-11T00:47:47.091+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): download_conceito_2021> on 2024-02-11 00:47:41.823216+00:00
[2024-02-11T00:47:47.106+0000] {standard_task_runner.py:60} INFO - Started process 176 to run task
[2024-02-11T00:47:47.111+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'censoEnade', 'download_conceito_2021', 'manual__2024-02-11T00:47:41.823216+00:00', '--job-id', '298', '--raw', '--subdir', 'DAGS_FOLDER/censoEnade.py', '--cfg-path', '/tmp/tmpmxn2fgw7']
[2024-02-11T00:47:47.113+0000] {standard_task_runner.py:88} INFO - Job 298: Subtask download_conceito_2021
[2024-02-11T00:47:47.206+0000] {task_command.py:423} INFO - Running <TaskInstance: censoEnade.download_conceito_2021 manual__2024-02-11T00:47:41.823216+00:00 [running]> on host 7e5b5e202339
[2024-02-11T00:47:47.330+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='NÃ©fi Fernandes' AIRFLOW_CTX_DAG_ID='censoEnade' AIRFLOW_CTX_TASK_ID='download_conceito_2021' AIRFLOW_CTX_EXECUTION_DATE='2024-02-11T00:47:41.823216+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-11T00:47:41.823216+00:00'
[2024-02-11T00:47:47.542+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host 'download.inep.gov.br'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

[2024-02-11T00:47:48.494+0000] {python.py:201} INFO - Done. Returned value was: C:/***-docker/dataset/conceito_enade_2021.xlsx
[2024-02-11T00:47:48.540+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=censoEnade, task_id=download_conceito_2021, execution_date=20240211T004741, start_date=20240211T004747, end_date=20240211T004748
[2024-02-11T00:47:48.610+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-11T00:47:48.646+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
